dev-packages/e2e-tests/package.json

```diff
@@ -23,12 +23,16 @@
     "jest": "^29.7.0",
     "react": "18.3.1",
     "react-native": "0.75.4",
+    "react-native-launch-arguments": "^4.0.2",
     "typescript": "4.9.5",
     "webdriverio": "^8.27.0"
   },
   "dependencies": {
     "minimist": "1.2.8",
     "semver": "7.6.3",
     "xcode": "3.0.1"
+  },
+  "peerDependencies": {
+    "react-native-launch-arguments": "^4.0.2"
```

Comment:
Q: adding it here and on peerDependencies? with that we got a reference to react-native-launch-arguments on the root yarn.lock

Reply:
The reason to add `react-native-launch-arguments` to peer dependencies is to indicate that the e2e-component package need this to be installed along side.

To be absolutely exact we should also include @sentry/react-native in here.

Reply:
In the root yarn.lock the dep is already based on the devDependencies.